apps/builder/app/builder/shared/html-editor.tsx

```diff
@@ -70,68 +70,47 @@ const wrapperStyle = css({
   position: "relative",
   // 1 line is 16px
   // set min 10 lines and max 20 lines
-  "--ws-code-editor-min-height": "160px",
-  "--ws-code-editor-max-height": "320px",
+  ...getMinMaxHeightVars({ minHeight: "160px", maxHeight: "320px" }),
 });
 
 export const HtmlEditor = forwardRef<
   HTMLDivElement,
-  {
-    readOnly?: boolean;
-    invalid?: boolean;
-    value: string;
-    title?: ReactNode;
-    onChange: (newValue: string) => void;
-    onBlur?: (event: FocusEvent) => void;
```

Comment:
we are just forwarding the props to the underlying component, there is really no need to duplicate all the types, I got hit by trying to find a property that wasn't listed here but is available in CodeEditor ....